### ISSUE TITLE
Enhancements to /settings

### ISSUE THREAD

**Issue by nstuyvesant** [DaftMonk/generator-angular-fullstack#1362]
A few minor items to consider to make the sample project a little more thorough...
- Perhaps consider add editing of the user's name on /settings rather than just password changes?
- In the /settings page, only show password changing (ng-if='user.provider == "local"')

**Comment by Awk34** [comment]
Yeah, that section has always been lacking, but lately we've been focusing on more pressing issues. I'd like to expand it eventually. Feel free to help out with a PR!
